OpenMTRDemo/Forms/BaseForm.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace OpenMTRDemo.Forms
```

Comment:
Wrong namespace.

Reply:
Yea I moved my forms into OpenMTRDemo/Forms/ . At some point we'll move all form components there.